src/app/pages/ProposalDetailsPage/hooks.ts

```diff
@@ -116,28 +117,54 @@ export const useVoteStats = (network: Network, proposalId: number): VoteStats =>
   }
 }
 
-export const useWantedVoteType = () =>
-  useTypedSearchParam<VoteType>('vote', 'any', {
+export const useVoteFiltering = (t: TFunction) => {
```

Comment:
How come you're pushing t through hooks instead of using useTranslation? Is there some limitation?

Reply:
There are no limitations that would apply here. We are sometimes doing this (passing `t` instead of _using_ it) in other situations, when the _rules of hooks_ might get in the way. (Ie. if the number of function calls might dynamically change while withing the render function if a given component.)  But it's not an issue here, so I have simplified the code..

At some point we could go over similar situations of the code, there might be other places that also can be simplified the same way.